internal/test/daemon/daemon_unix.go

```diff
@@ -21,7 +21,7 @@ func cleanupNetworkNamespace(t testingT, execRoot string) {
 	// new exec root.
 	netnsPath := filepath.Join(execRoot, "netns")
 	filepath.Walk(netnsPath, func(path string, info os.FileInfo, err error) error {
-		if err := unix.Unmount(path, unix.MNT_FORCE); err != nil {
+		if err := unix.Unmount(path, unix.MNT_DETACH); err != nil && err != unix.EINVAL && err != unix.ENOENT {
```

Comment:
Looks like this change was already merged in https://github.com/moby/moby/commit/73baee2dcf546b2561bdd9a500b0af08cb62b1be (as part of https://github.com/moby/moby/pull/38127)